style.css

```diff
@@ -0,0 +1,124 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100%;
+  min-height: 100vh;
+  background-color: #f0f0f0;
+}
+
+header {
+  background-color: #fff;
+}
+
+.ambuger-menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.bar {
+  display: block;
+  width: 33px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #162ae3;
+}
+
+.nav-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem;
+}
+
+section {
+  padding: 5rem 1.25rem 6rem;
+  background-color: #fff;
+}
+
+section:nth-of-type(1) {
+  border-bottom-left-radius: 80px;
+  background-image: url("./image/hero_bg.png");
```

Comment:
- [ ] Kindly make sure this background display as expected as you see in the picture above the background image is not visible please make  sure you  fix that, unsure that you use the right path and correct name of the image :+1: